src/app/core/store/search/search.action.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Lumeer: Modern Data Definition and Processing Platform
+ *
+ * Copyright (C) since 2017 Answer Institute, s.r.o. and/or its affiliates.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import {Action} from '@ngrx/store';
+
+export enum SearchActionType {
+
+  GET_SELECTED_DOCUMENTS = '[Search Documents] Get selected',
```

Comment:
You don't need an action for getting selected documents since you don't have to load it from the server and so you can directly read it from the store.

Reply:
Yes yes you are right. This action is unused. I will delete it..